packages/http-client/src/ceramic-http-client.ts

```diff
@@ -7,21 +7,43 @@ import { TileDoctypeHandler } from "@ceramicnetwork/doctype-tile"
 import { Caip10LinkDoctypeHandler } from "@ceramicnetwork/doctype-caip10-link"
 import DocID from '@ceramicnetwork/docid'
 
-const CERAMIC_HOST = 'http://localhost:7007'
 const API_PATH = '/api/v0'
+const CERAMIC_HOST = 'http://localhost:7007'
+
+/**
+ * Default Ceramic client configuration
+ */
+export const DEFAULT_CLIENT_CONFIG: CeramicClientConfig = {
+  docSyncEnabled: false,
+  docSyncInterval: 5000,
```

Comment:
What about `docSyncIntervalMs`?  I'm not a fan of numeric arguments that are ambiguous about what unit they take.  Time arguments are especially prone to bugs I've found.  In a language with a stronger type system I like to use the type system to handle this (i.e. take a Milliseconds type that does not support any implicit conversions from numeric constants), but in lieu of that, encoding the unit into the name is the next best option I believe.

I know this PR was already merged, but since this argument is user-facing I figured it was still worth raising in case we want to put together a small change before releasing and getting locked in.  @oed @simonovic86 @PaulLeCam

Reply:
@stbrody hm, I guess it's enough that we can document the unit for the variable. I've opened the story for documenting the http client [537](https://github.com/ceramicnetwork/js-ceramic/issues/537).

Reply:
@simonovic86 why not rename the field to include the unit?  Then it becomes self-documenting and makes it immediately obvious at all call sites what the value represents